salt/modules/win_wusa.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+'''
+Microsoft Update files management via wusa.exe
+
+:maintainer:    Thomas Lemarchand
+:platform:      Windows
+:depends:       PowerShell
+
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+
+# Import salt libs
+import salt.utils.platform
+
+log = logging.getLogger(__name__)
+
+# Define the module's virtual name
+__virtualname__ = 'win_wusa'
+
+
+def __virtual__():
+    '''
+    Load only on Windows
+    '''
+    if not salt.utils.platform.is_windows():
+        return False, 'Only available on Windows systems'
+
+    powershell_info = __salt__['cmd.shell_info'](shell='powershell', list_modules=True)
```

Comment:
Pls do not set `list_modules=True` as your not using the extra data return. It takes time for it to scan the file system to determine a list of modules available.  The more modules the longer it takes.  It should only be used if a module is dependent on a particular module the user might need to install or old version of powershell did not have.

Reply:
You're right. It's fixed.